tests/phpunit/class-wc-stripe-test-suite-loader.php

```
@@ -0,0 +1,159 @@
+				throw new Exception(
+					sprintf(
+						'Method %s::suite() declared in %s is not static',
+						$suite_class_name,
+						$suite_class_file
+					)
+				);
+			}
+		}
+
+		return $class;
+	}
```

PHPMD (2.15.0)

[warning] The method load() has a Cyclomatic Complexity of 21. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)